docs/sdks/server-sdks/dotnet/quickstart.mdx

```diff
@@ -0,0 +1,224 @@
+| `GetStringAssignment()` | String |
+| `GetBooleanAssignment()` | Boolean |
+| `GetJSONAssignment()` | JSON object |
+| `GetIntegerAssignment()` | Integer |
+| `GetNumericAssignment()` | Float |
+
+### Using Assignments
+
+After receiving an assignment, your application should implement logic to modify the user experience accordingly:
+
+```csharp
+var variation = eppoClient.GetStringAssignment(
+    flagKey,
+    subjectKey,
+    subjectAttributes,
+    defaultValue
+);
+
+// Render different components based on assignment
+switch(variation) 
+{
+    case "landing-page-a":
+        return RenderLandingPageA();
+    case "landing-page-b":
+        return RenderLandingPageB();
+    default:
+        return RenderLandingPageC();
+}
+```
+
+## Experiments
+
+While feature flags are useful, they do not send you any information about how your users are interacting with the feature. Experiments provide a way to collect data about these interactions using whichever logging and data warehousing system you prefer.
+
+To log events through the SDK, you need to define a logger that implements the `IAssignmentLogger` interface:
+
+<Tabs>
+<TabItem value="console" label="Console">
+
+```csharp
+internal class AssignmentLogger : IAssignmentLogger
+{
+    public void LogAssignment(AssignmentLogData assignmentLogData)
+    {
+        Console.WriteLine(assignmentLogData);
+    }
+}
+```
+
+</TabItem>
+<TabItem value="segment" label="Segment">
+
+```csharp
+internal class SegmentLogger : IAssignmentLogger
+{
+    private readonly Analytics analytics;
+
+    public SegmentLogger(Analytics analytics)
+    {
+        this.analytics = analytics;
+    }
+
+    public void LogAssignment(AssignmentLogData assignmentLogData)
+    {
+        analytics.Track("Eppo Randomization Assignment", assignmentLogData);
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+In a production application, you would want to replace the `Console.WriteLine()` statement with an actual logging system. We have documentation on how to set up logging with multiple popular data warehouses and logging systems in the [Assignment page](/sdks/server-sdks/dotnet/assignments/#logging-data-to-your-data-warehouse).
+:::
+
+## Contextual Bandits
+
+For bandits, you'll need both an assignment logger and a bandit logger:
+
+```csharp
+internal class BanditLogger : IAssignmentLogger
+{
+    public void LogAssignment(AssignmentLogData assignmentLogData)
+    {
+        Console.WriteLine(assignmentLogData);
+    }
+
+    public void LogBanditAction(BanditLogEvent banditLogEvent)
+    {
+        Console.WriteLine(banditLogEvent);
+    }
+}
+```
+
+### Query the bandit for actions
+
+Instead of making simple assignments with a bandit, you will want to query the bandit for actions. 
+
+
+```csharp
+var subjectAttributes = new Dictionary<string, object?>()
+{
+    ["age"] = 30,
+    ["country"] = "uk",
+    ["pricingTier"] = "1"
+};
+
+var actions = new Dictionary<string, IDictionary<string, object?>>()
```

Comment:
We have a helper class, `ContextAttributes` which makes this typing a little simpler. Check out
https://github.com/Eppo-exp/dot-net-server-sdk/blob/89f65bb5779a7293beff878534381dcc871b03cd/dot-net-sdk/EppoClient.cs#L334

Reply:
Alright. I'm updating this code example to be more similar to the example in the SDK's code comments. I'll update the subject and the actions with context attributes. 

@typotter Would it make sense to full docs of the ContextAtrribute as part of the Bandits doc page?

Reply:
Nevermind, it looks like we explain that type there and it's a simple method.